track_and_trace/asset_client/src/services/auth.js

```diff
@@ -231,17 +232,16 @@ const AuthService = {
 
     /**
      * Creates a user then submits a transaction to the blockchain
-     * 
+     *
      * The function is a (Signer) => Promise, where the promise is resolved when the transaction completes
      */
     createUser: (user, submitTransactionFn) => {
-        let userCreate = pluck(user, 'username', 'password', 'email', 'name')
+        let userCreate = pluck(user, 'password', 'email')
```

Comment:
is the database also updated to reflect this?

Reply:
Yes, in a previous PR.